src/arch/aarch64/kernel/mod.rs

```diff
@@ -91,15 +91,7 @@ pub fn get_processor_count() -> u32 {
 }
 
 pub fn args() -> Option<&'static str> {
-	let dtb = unsafe {
-		hermit_dtb::Dtb::from_raw(ptr::with_exposed_provenance(
-			boot_info().hardware_info.device_tree.unwrap().get() as usize,
-		))
-		.expect(".dtb file has invalid header")
-	};
-
-	dtb.get_property("/chosen", "bootargs")
-		.map(|property| str::from_utf8(property).unwrap())
+	None
```

Comment:
Couldn't we throw out these functions while we are on it?

Reply:
We can do away with these architecture-specific bootargs once we have migrated all platforms to FDT bootargs and do the bootinfo version jump. We still have the bootargs in the bootinfo for x86-64 QEMU and Linux boot.